python/cuda_parallel/cuda/parallel/experimental/iterators/_iterators.py

```diff
@@ -155,12 +155,12 @@ def impl(ptr, offset):
     return impl
 
 
-class RawPointerType(IteratorKind):
+class RawPointerKind(IteratorKind):
```

Comment:
I hope reviewers don't mind that this change is piggybacking on the PR. Happy to move it to a separate PR if folks are opposed.

Reply:
(it's just renaming a couple of Python classes, which I forgot to do in a previous PR)